tests/e2e/test_misc_commands.py

```diff
@@ -62,9 +62,9 @@ def test_init_ask_existing(protostar_bin: Path):
 
```

Comment:
Please add a test case that checks the Cairo version format.

Reply:
done